_docs/cibuildSetup.md

```diff
@@ -0,0 +1,62 @@
+# CI BUILD SETUP
+
+## Docker build assessment
+
+### Option 1 - Travis driven
+
+How?
+
+* travis could do (cibuild script, using others):
+    * running unit (go tests)
+    * compiling (go build)
+    * end to end / cmdline tests (to be developed)
+    * docker build
+    * end to end / cmdline tests using docker image (to be developed)
+    * docker push
+    * updating readme -help (to be developed)
+
+Advantages?
+* Just a single build/deploy logic in travis
+* Easy to reproduce locally
+
+Problems?
+* We will need to maintain the description in dockerhub manually
```

Comment:
this https://stackoverflow.com/questions/34710513/how-can-i-update-full-description-on-docker-hub-automatically doesn't solve this?

Reply:
Did you read it? It actually describes option 2 :)